### DISCUSSION TITLE
Can you select multiple notes in Link Creator?

### DISCUSSION THREAD

**Discussion by troyjcross** [windingwind/zotero-better-notes#1038] [Q&A]
I was wondering if it was possible (or will be possible in future) to select multiple notes at one time for inserting into a note via the "Link Creator" UI? 

At present, if I do manage to select multiple notes in Step 1, only the first note gets inserted to the position in Step 2.

**Comment by windingwind** [comment]
Theoretically, the `link to` can support multiple selection, while the `mention in` should not. I'll create a ticket for this. Thanks for the report.
